### ISSUE TITLE
 question;

### ISSUE THREAD

**Issue by lujunxin** [lengoanhcat/mcgm_halide#1]
Hello, thank you very much for your project. I have some problems when it runs the program. The details are as follows: I generated mcgmoppticalflow. O. But when I run runmcgmoppticalflow, I call 
Eval (['mcgmoppticalflow (indata, filterthreshold, divisionthreshold, divisionthreshold2, speedthreshold, ', outvar,'); '); 
in this sentence, matlab displays the following error:
![2019-11-19 21-18-32屏幕截图](https://user-images.githubusercontent.com/45283735/69149870-60d35180-0b12-11ea-8915-e50da0a48d88.png)

I wonder if you can give me an answer. By the way, my mcgmoppticalflow. O is generated under / TMP / mex halide / on the Linux system,
![2019-11-19 21-20-32屏幕截图](https://user-images.githubusercontent.com/45283735/69149885-6e88d700-0b12-11ea-9f2f-80ff85db7b13.png)

**Comment by lengoanhcat** [comment]
@lujunxin : could you help translate the error from Chinese to English ?

**Comment by lujunxin** [comment]
> @lujunxin : could you help translate the error from Chinese to English ?

Undefined variables and functions：mcgmopticalflow

**Comment by lengoanhcat** [comment]
@lujunxin : it complains that mcgmOpticalFlow ( a compiled file ) could not be found. Have you successfully compile mcgmOpticalFlow ?

**Comment by lujunxin** [comment]
yes, i have compiled ,the result in the /tmp/mex-halide  ------------------ 原始邮件 ------------------
发件人: "LE NGO ANH CAT"<notifications@github.com>
发送时间: 2019年11月20日(星期三) 中午11:09
收件人: "lengoanhcat/mcgm_halide"<mcgm_halide@noreply.github.com>;
抄送: "lujunxin"<792902802@qq.com>;"Mention"<mention@noreply.github.com>;
主题: Re: [lengoanhcat/mcgm_halide]  question; (#1)



@lujunxin : it complains that mcgmOpticalFlow ( a compiled file ) could not be found. Have you successfully compile mcgmOpticalFlow ?
 
&mdash;

**Comment by lengoanhcat** [comment]
Try addpath('/tmp/mex-halide') b4 running the matlab code. MATLAB might not be able to find the executive file.

**Comment by lujunxin** [comment]
Thank you very much. I'll try again

**Comment by lujunxin** [comment]
Sorry to bother you.I am confused about some of the formulas in the code.Do you have any reference papers for this code?Are there any papers or books that i can deduce by myself?Thank you very much!

**Comment by lengoanhcat** [comment]
"A color neuromorphic approach for motion estimation". https://dl.acm.org/citation.cfm?id=1899240 This is a good description of the technique.
